### PR TITLE
Feat/bot

### PR DESCRIPTION
**Pull request by igbopharaoh**
- Add initial setup for bot

### CODE REVIEW

File: client/fresheyes-bot/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        .reduce((acc, curr) => {
+          const key = curr.line;
+
+          const group = acc[key] ?? [];
+
+          return { ...acc, [key]: [...group, curr] };
+        }, {});
+
+      await Promise.all(
+        Object.entries(groupComments).map(async ([k, v]) => {
```
Comment (Extheoisah) [inline client/fresheyes-bot/src/index.ts]: what's `k` and `v`. Lets use descriptive variable assignments

File: client/fresheyes-bot/tsconfig.json
Comment (Extheoisah) [inline client/fresheyes-bot/tsconfig.json:1]: Pls remove commented unused code

File: client/fresheyes-bot/app.yml
Comment (Extheoisah) [inline client/fresheyes-bot/app.yml:1]: pls remove commented usused code

File: client/fresheyes-bot/src/index.ts (hunk 1 above)
Comment (igbopharaoh) [reply client/fresheyes-bot/src/index.ts]: I will make comments more descriptive

**Review by Extheoisah — APPROVED** [review]

**Pull request merged**